.github/workflows/test.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           python -m pip install flake8
       - name: flake8
         run: |
-          flake8 .
+          flake8 . --ignore E501,W503
```

Comment:
The repo has a couple of places where it configures flake8

https://github.com/torchbox/wagtail-content-import/blob/master/.flake8
https://github.com/torchbox/wagtail-content-import/blob/master/setup.cfg#L1-L5

would be good to standardise on one so this type of change is not needed

Reply:
Hi @zerolab ,
I have deleted the .flake8 file, and just transferred the remaining settings on setup.cfg. I have also reverted the change above. Thank you!